翻译客服文档 (#833)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/tap-support/features/roles.mdx

---

> title: TapSupport Service Privilege Control

Reviewer: ```suggestion
title: TapSupport Authority Control
```

> sidebar_label: Customer Service Privileges

Reviewer: ```suggestion
sidebar_label: Customer Service Authorities
```

sidebar_position: 2
---

> The Customer Service module has a separate permissions system from the Developer Center section. Permissions consist of two parts: roles and scopes.

Reviewer: ```suggestion
The Customer Service module has a separate authority system from the Developer Center section. Authorities consist of two parts: role and scope.
```

- Role: determines which functions and actions a user can use;
- Scope: determines which work orders a user can access.

### Roles

> User roles are categorized into the following according to their different responsibilities:

Reviewer 2: ```suggestion
User roles are categorized into the following categories according to their different responsibilities:
```

| user role                  | remit                                                                                                                        | access path                | billing    |
| -------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | -------------------------- | ---------- |

> | Manufacturer Administrator | Maintenance services and security-related configurations                                                                     | Developer Center           | not billed |

Reviewer 2: ```suggestion
| Super Administrator | Maintenance services and security-related configurations                                                                     | Developer Center           | not billed |
```

| Administrator              | daily management<br/><small>Has all configuration permissions except for service and security related configurations</small> | Customer Service Workbench | billing    |
| Customer Service           | Communicate directly with players and handle player requests                                                                 | Customer Service Workbench | billing    |

> Vendor Administrators are Tap users with customer service privileges in the Developer Center. Users in other roles are independent users of the customer service system who log in to use the customer service module through the Customer Service Workbench, which is separate from the Developer Center, and are collectively referred to as "Members". Members can be added by vendor administrators in the Developer Center or by administrators in the Customer Service Workbench. A member can have one or more user roles, and vendor administrators or administrators can modify the roles of members.

Reviewer 2: ```suggestion
Super Administrators are Tap users with TapSupport authorities in the Developer Center. Users in other roles are independent users of the customer service system who log in to use the TapSupport module through the TapSupport Workbench, which is separate from the Developer Center, and are collectively referred to as "Members". Members can be added by super administrators in the Developer Center or by administrators in the TapSupport Workbench. A member can have one or more user roles, and super administrators or administrators can modify the roles of members.
```

The detailed correspondence between roles and permissions is as follows:

> |                             |                                                                              |           Manufacturer Administrator       |            janitors             | customer services | collaborators |          developers           |

Reviewer 2: ```suggestion
|                             |                                                                              |           Super Administrator       |            Administrator             | Customer Services | Collaborators |          Developers           |
```

| --------------------------- | ---------------------------------------------------------------------------- | :----------------------------: | :----------------------------: | :--: | :----: | :-------------------------: |
| Services and Security       | Enabling, disabling services                                                               |               ✔️               |                                |      |        |                             |

> |                             | Configure a custom domain name                                                             |               ✔️               |                                |      |        |                             |

Reviewer 2: ```suggestion
|                             | Configuring a custom domain name                                                             |               ✔️               |                                |      |        |                             |
```

> | user management             | Query member list and information                                                           |               ✔️               |               ✔️               |  ✔️  |   ✔️   |             ✔️              |

Reviewer 2: ```suggestion
| User Management             | Querying member list and information                                                           |               ✔️               |               ✔️               |  ✔️  |   ✔️   |             ✔️              |
```

> |                             | Management members<br/><small>Add, disable, and modify profiles</small>                             |               ✔️               |               ✔️               |      |        |                             |

Reviewer 2: ```suggestion
|                             | Manage members<br/><small>Add, disable, and modify profiles</small>                             |               ✔️               |               ✔️               |      |        |                             |
```

> |                             | Management members<br/><small>Add, disable, and modify profiles</small>                             |               ✔️               |               ✔️               |      |        |                             |

Reviewer 2: ```suggestion
|                             | Managing members<br/><small>Adding, disabling, and modifying profiles</small>                             |               ✔️               |               ✔️               |      |        |                             |
```

> |                             | Check Player Profile                                                          |                                |               ✔️               |  ✔️  |   ✔️   |             ✔️              |

Reviewer 2: ```suggestion
|                             | Checking Player Profile                                                          |                                |               ✔️               |  ✔️  |   ✔️   |             ✔️              |
```

|                             | Managing Players<br/><small>Adding and modifying information</small>                               |                                |               ✔️               |  ✔️  |        |                             |

> | work order                  | Access to work orders<br/><small>- Inquire and view work orders<br/>- Submit an internal message</small>              |                                |               ✔️               |  ✔️  |   ✔️   |             ✔️              |

Reviewer 2: ```suggestion
| Tickets                  | Access to Tickets<br/><small>- Inquiring and viewing tickets<br/>- Submitting an internal message</small>              |                                |               ✔️               |  ✔️  |   ✔️   |             ✔️              |
```

> |                             | Modifying Work Order Properties                                                               |                                |               ✔️               |  ✔️  |        |                             |

Reviewer 2: ```suggestion
|                             | Modifying Tickets Properties                                                               |                                |               ✔️               |  ✔️  |        |                             |
```

> |                             | Reply to the player                                                                   |                                |                                |  ✔️  |        |                             |

Reviewer 2: ```suggestion
|                             | Replying to the player                                                                   |                                |                                |  ✔️  |        |                             |
```

> | repository                  | check up                                                                        |                                |               ✔️               |  ✔️  |   ✔️   |             ✔️              |

Reviewer 2: ```suggestion
| Knowledge Base                  | Checking up                                                                        |                                |               ✔️               |  ✔️  |   ✔️   |             ✔️              |
```

> |                             | managerial<br/><small>Adding, modifying content and posting status</small>                             |                                |               ✔️               |  ✔️  |        |                             |

Reviewer 2: ```suggestion
|                             | Management<br/><small>Adding, modifying content and posting status</small>                             |                                |               ✔️               |  ✔️  |        |                             |
```

> | setting                     | Content Setting<br/><small>- Products and Classification<br/>- Work order fields, forms<br/>- dynamic content</small> |                                |               ✔️               |      |        | ✔️ <br/><small>read only</small> |

Reviewer 2: ```suggestion
| Setting                     | Content Setting<br/><small>- Products and Classification<br/>- Fields and forms of Tickets<br/>- Dynamic content</small> |                                |               ✔️               |      |        | ✔️ <br/><small>read only</small> |
```

> |                             | Customer Service Settings<br/><small>- cluster<br/>- ignitor<br/>- Notification channel management</small>                                         |                                |               ✔️               |      |        |                             |

Reviewer 2: ```suggestion
|                             | TapSupport Settings<br/><small>- Cluster<br/>- Trigger<br/>- Notification channel management</small>                                         |                                |               ✔️               |      |        |                             |
```

|                             | Development Settings<br/><small>- Player Authentication Methods<br/>- Customized domain names (read only)</small>      |                                |               ✔️               |      |        |             ✔️              |

> | audits                      | Audit log                                                                     | ✔️<br/><small>Manufacturers' section</small> | ✔️<br/><small>Membership component</small> |      |        |                             |

Reviewer 2: ```suggestion
| Audits                      | Audit log                                                                     | ✔️<br/><small>Manufacturers' section</small> | ✔️<br/><small>Membership component</small> |      |        |                             |
```

### Scope

> If a user has the "Access Work Orders" permission, there are three different scopes of work orders that he/she can access as follows:

Reviewer 2: ```suggestion
If a user has the "Access Tickets" permission, there are three different scopes of work orders that he/she can access as follows:
```

> - All Work Orders

Reviewer 2: ```suggestion
- All Tickets
```

> - All work orders

Reviewer 2: ```suggestion
- Only Tickets for its group
```

> - All Work Orders Work Orders for their group only Work Orders assigned to this user only

Reviewer 2: ```suggestion
- Only Tickets assigned to this user only
```
